refarch-gateway/README.md

```diff
@@ -41,17 +41,40 @@ Beside the default behaviour there are some special route prefixes which are han
-| `REFARCH_SECURITY_CSRFWHITELISTED_<index>` (optional)    | List of routes to disable csrf protection for.     | `/example/**`                                                           |
-| `INFO_APPSWITCHER_URL` (optional)                        | App switcher url for usage in refarch frontend.    | `https://appswitcher.muenchen.de`                                       |
+| Var                                                       | Description                                        | Example                                                                 |
+|-----------------------------------------------------------|----------------------------------------------------|-------------------------------------------------------------------------|
+| `SPRING_PROFILES_ACTIVE`                                  | See profiles                                       | `local,hazelcast-local`                                                 |
+| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_ID`                  | Id of a route definition.                          | `backend`                                                               |
+| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_URI`                 | The uri to route to if this route matches.         | `http://backend-service:8080/`                                          |
+| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_PREDICATES_<index>`  | Route predicates i.e. matcher.                     | `Path=/api/backend-service/**`                                          |
+| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_FILTERS_<index>`     | List of filters applied to the route.              | `RewritePath=/api/backend-service/(?<urlsegments>.*), /$\{urlsegments}` |
+| `ALLOWED_ORIGINS_PUBLIC` (optional)                       | List of urls allowed as origin for public routes.  | `https://*.muenchen.de,http://localhost:*`                              |
+| `ALLOWED_ORIGINS_CLIENTS` (optional)                      | List of urls allowed as origin for clients routes. | `https://*.muenchen.de,http://localhost:*`                              |
+| `REFARCH_SECURITY_CSRFWHITELISTED_<index>` (optional)     | List of routes to disable csrf protection for.     | `/example/**`                                                           |
+| `INFO_APPSWITCHER_URL` (optional)                         | App switcher url for usage in refarch frontend.    | `https://appswitcher.muenchen.de`                                       |
+
+### Security
+
+For authentication via SSO, OAuth2 needs to be configured.
+See following example or the [according Spring documentation](https://docs.spring.io/spring-security/reference/servlet/oauth2/index.html).
+
+```yaml
+spring:
+  # optional with profile "no-security"
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: "https://sso.muenchen.de/auth/realms/muenchen.de"
```

Comment:
Should be maybe provide a more general domain here? like `sso.domain.com/auth/...`?

Reply:
Done